actions/create_file.yaml

```diff
@@ -45,3 +45,15 @@ parameters:
     type: "string"
     description: "If omitted this will be filled in with committer information. If passed, you must specify both a name and email. Expected format: FirstName LastName <email@address>"
     required: false
+
+  api_user:
+    type: "string"
+    description: "The"
```

Comment:
I don't think the description of this parameters was finished, as it just states "The"